src/engine/DrawManager.java

```diff
@@ -2725,25 +2725,34 @@ public void animateLoading(int x, int y){
 	 * @param separationLineHeight
 	 * @param lives
 	 */
-	public void drawBackground(final Screen screen, int separationLineHeight, int lives){
+	public void drawBackground(final Screen screen, int separationLineHeight, int lives, int level){
```

Comment:
[P3] It's nice to be able to add levels to the parameter so that it can be understood simply as "a function that draws a background based on the level".